src/Compilers/CSharp/Portable/FlowAnalysis/VariablesDeclaredWalker.cs

```diff
@@ -137,5 +137,16 @@ public override BoundNode VisitQueryClause(BoundQueryClause node)
 
             return base.VisitQueryClause(node);
         }
+
+        protected override void VisitLvalue(BoundLocal node)
+        {
+            if (!node.WasCompilerGenerated && node.Syntax.Kind() == SyntaxKind.Argument &&
```

Comment:
Right, somehow I lost it. I'll make sure to add tests for that. And address the issue in the next PR.

---
In reply to: [63744241](https://github.com/dotnet/roslyn/pull/11393#discussion_r63744241) [](ancestors = 63744241)